pyanaconda/modules/common/structures/realm.py

```diff
@@ -71,3 +72,12 @@ def join_options(self) -> List[Str]:
     @join_options.setter
     def join_options(self, options: List[Str]):
         self._join_options = options
+
+    @property
+    def realm_discovered(self) -> Bool:
```

Comment:
I would rename to  `discovered`. It is clear, that it is related to a realm.

Reply:
Yep, renamed. :)